mantidproject/mantid
# LoadVesuvio read 2 and 3 period data

Fixes #14735
- 2 and 3 period data should now be successfully read and processed by LoadVesuvio
- The normalisation window has been change for back scattering to `600-700` from `410-430`
- Tests have been updated for the normalisation change but no new test cases can be added due to not having appropriate data at this time #14924 
# To Test
- Ensure the tests pass and the updates cover the changes made here
- Test LoadVesuvio with 2 and 3 period data to ensure correctness
  - 3 period = `20598`
  - 2 period = `21298-21302`
